src/main.test.ts

```diff
@@ -0,0 +1,5 @@
+describe("Our bot", () => {
```

Comment:
We should figure out why the tests are not found anymore instead of working around like this.

Reply:
Oh, we don't have any tests anymore because argument parsing is gone. You can remove this file and comment out the workflow file instead.